ofctrl/fgraphFlow.go

```diff
@@ -610,27 +604,42 @@ func getRangeEnd(rng *openflow13.NXRange) uint16 {
 	return rng.GetOfs() + rng.GetNbits() - 1
 }
 
-func merge(regs []*NXRegister) *NXRegister {
-	if len(regs) == 1 {
-		return regs[0]
+func getStartFromMask(mask uint32) uint16 {
+	var count uint16
+
+	if mask == 0 {
+		return 0
+	}
+
```

Comment:
It might return 1 if mask==0x1, but we actually want to mask bit 0.

Reply:
Actully, this check just to eliminate doubt whether L614 go into infinite loop once mask is 0. From L625, we know mask cannot be 0.
 If mask is 0x1, it represents we are using range[0:0], we should return 0 as start.  In merge(), we will check reg.Range to compute mask if mask is 0. For mask is 0 and range is nil, we we can think we are using full range.

Reply:
But the reg.Data should left shift 1bit  in L625, and this is not what we want (the data should shift 0bit as expect), right?

Reply:
My bad, your code change is correct.